### DISCUSSION TITLE
Testing v1.0.0 and it seems mealie requires registration to view recipies. Can it be turned off?

### DISCUSSION THREAD

**Discussion by DoTheEvo** [mealie-recipes/mealie#2241] [Q&A] [answered]
Test recipe seems to be set to public.
I tried with disabled registration too.

Is mealise going to lose the function to view recipies without registration? 
Cuz I am literally switching from tandoor exactly because [that one is missing this feature](https://github.com/TandoorRecipes/recipes/issues/49#issuecomment-1199849331).

I will still switch to mealie as the current version works how I want it and it seems it should work fine for my use.
But I would of course prefer if major feature would not get removed...

Or maybe I just missed some setting that would allow non-registred users see public recipies/cookboosk.

**Comment by hay-kot** [comment, marked as answer]
It's on the roadmap, you can track progress here

- https://github.com/hay-kot/mealie/issues/1326
